cypress/integration/reset.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Reset and Complete Password', () => {
+    it("type send a reset an email address", () => {
+        cy.visit('/reset');
+        cy.get("#email")
+          .type("andela@andela.com")
+          .should("have.value", "andela@andela.com");
+        cy.get('#sendEmail').click();
+        cy.wait(1000);
+        cy.get('.Toastify').should('not.be.visible');
+      });
+      it("should be able to give an error message when link is invalid", () => {
+        cy.visit('/completReset/token');
+        cy.get("#newPassword")
+          .type("Password@123")
+          .should("have.value", "Password@123");
+        cy.get("#confirmNewPassword")
+          .type("Password@123")
+          .should("have.value", "Password@123");
+        cy.get('#btn-send').click();
+        cy.wait(1000);
+        cy.get('.Toastify').should('not.be.visible');
```

Comment:
From the description of this test, can't see any `should` or `except` assertion for the displayed error message